gno.land/pkg/sdk/vm/keeper.go

```diff
@@ -336,7 +336,7 @@ func (vm *VMKeeper) AddPackage(ctx sdk.Context, msg MsgAddPackage) (err error) {
 	}
 	creatorAcc := vm.acck.GetAccount(ctx, creator)
 	if creatorAcc == nil {
-		return std.ErrUnknownAddress(fmt.Sprintf("account %s does not exist", creator))
+		return std.ErrUnknownAddress(fmt.Sprintf("account %s has not been initialized. does it have coins?", creator))
```

Comment:
```suggestion
		return std.ErrUnknownAddress(fmt.Sprintf("account %s does not exist or has not been initialized. does it have coins?", creator))
```
We still want the error to make sense if an address doesn't exist at all.